### PR TITLE
I18n sync HOC 09/20

### PR DESCRIPTION
**Pull request by Hamms**
First full HOC sync of 2018!

### CODE REVIEW

File: i18n/locales/nl-NL/hourofcode/how-to/afterschool.md [hunk 1]
```diff
@@ -7,19 +7,21 @@ nav: how_to_nav
...
+### Word lid van de beweging en laat een groep studenten kennismaken met hun eerste 'Hour of Code' met deze stappen. Het 'Hour of Code' is makkelijk te geven - zelfs voor beginners! If you'd like an extra set of hands to help out, you can find a [local volunteer](%= codeorg_url('/volunteer/local') %) to help run an Hour of Code in your after-school class or club.
+
+* * *
 
 ## 1. Watch this how-to video <iframe width="500" height="255" src="//www.youtube.com/embed/SrnvvWDm73k" frameborder="0" allowfullscreen mark="crwd-mark"></iframe> 
 
 ## 2. Kies een tutorial
 
-Wij bieden een verscheidenheid van [leuke, urenlange tutorials](%= resolve_url('/learn') %) voor deelnemers van alle leeftijden, gemaakt door een scala aan partners. [Probeer ze uit](%= resolve_url('/learn') %)
+We provide a variety of [fun, hour-long tutorials](%= resolve_url('/learn') %) for participants all ages, created by a variety of partners. [Probeer ze uit](%= resolve_url('/learn') %)
```
Comment (tanyaparker) [inline i18n/locales/nl-NL/hourofcode/how-to/afterschool.md:18]: We lost some translations. How do I check if there was a change in the English string? In Crowdin I still see the translation and it says 100% match as if the English string didn't change.

Comment (tanyaparker) [reply i18n/locales/nl-NL/hourofcode/how-to/afterschool.md:18]: Maybe this is more a question for Jake, do we want to pre-translate the project when we make extensive copy changes to the site?

Comment (Hamms) [reply i18n/locales/nl-NL/hourofcode/how-to/afterschool.md:18]: This is probably the same as the Spanish fixes you did recently; there isn't a great way that I know of to verify changes in the English string, but you can [view the translation history](https://crowdin.com/project/hour-of-code/activity_stream?translation_id=169501&lang=36) for the string to see that it's "new" as of a month ago

Comment (tanyaparker) [reply i18n/locales/nl-NL/hourofcode/how-to/afterschool.md:18]: Gotcha, I'll follow up with Jake.

**Pull request merged**